### PR TITLE
Return DASH manifest as data URI

### PR DESCRIPTION
**Pull request by TpmKranz**
This is a workaround for cases when the manifest is not a JSON object but a b64-encoded DASH manifest. The `data` scheme allows to keep the interface of `get_file_url` unchanged. Clients that can't deal with data URIs would've choked on the `JSONDecodeError` anyway and those that do will just consume them like URLs. Tested with ffmpeg/ffplay and Clementine, both of which don't mind playing data URIs. MPV, for example, doesn't like them.

Linting failed due to files unchanged by me, so I had to turn it off. Sorry about that.

### CODE REVIEW

**Review by wvffle — APPROVED** [review]

File: tidal_async/api.py [hunk 1]
```diff
@@ -307,12 +303,16 @@ async def get_file_url(
...
+        playback_info = await self._playbackinfopostpaywall(preferred_quality)
         quality = AudioQuality(playback_info["audioQuality"])
 
         if quality < required_quality:
             raise InsufficientAudioQuality(f"Got {quality} for {self}, required audio quality is {required_quality}")
 
+        try:
+            manifest = json.loads(base64.b64decode(playback_info["manifest"]))
+        except json.decoder.JSONDecodeError:
+            return "data:application/dash+xml;base64,{}".format(playback_info["manifest"])
```
Comment (JuniorJPDJ) [inline tidal_async/api.py:315]:
```suggestion
            return f'data:application/dash+xml;base64,{playback_info["manifest"]}'
```

**Review by JuniorJPDJ — COMMENTED** [review]
Very good idea btw ;)
Thanks for your contribution.

**Comment by JuniorJPDJ** [comment]
> Linting failed due to files unchanged by me, so I had to turn it off. Sorry about that.

Thanks, good to know. I'll take care about it in a moment.

**Comment by JuniorJPDJ** [comment]
@TpmKranz could you rebase on top of master?
linter and test errors should be fixed

**Comment by JuniorJPDJ** [comment]
Also would be cool if you could reword commit message to conform with [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

If you can't do it I can, just say ;)

**Pull request merged**